vite.config.mts

```diff
@@ -72,7 +72,7 @@ export default defineConfig((config) => ({
 
 // Workaround for react-virtualized with vite
 // https://github.com/bvaughn/react-virtualized/issues/1632#issuecomment-1483966063
-function reactVirtualized(): PluginOption {
+function reactVirtualized() {
```

Comment:
Why did you remove the type? TypeScript didn't infer correctly in IDE inside the function.

Reply:
ah I understand
```
/home/chuinetri/Projects/__git/_gridsuite/commons-ui/vite.config.mts
  10:15  warning  'PluginOption' is defined but never used  no-unused-vars
```

It's a error of eslint, the type is used 😕  
I add a ignore on my branch